package.json

```diff
@@ -18,38 +18,30 @@
   },
   "homepage": "https://github.com/uwblueprint/building-up-server#readme",
   "devDependencies": {
-    "@types/node": "^14.10.1",
```

Comment:
I removed the unused dependencies but someone should double-check to ensure I didn't remove anything important

Reply:
@jzxhuang do you think we'll be using any of these? For instance `@types/cors`? I'm still not entirely sure what all of these packages are used for at the moment

Reply:
Any `@types/**` gives type declarations for Typescript

There's no need to include them since we're not using Typescript